app.js

```diff
@@ -0,0 +1,170 @@
+const express = require('express');
+// const { where } = require('sequelize/types');
+const { Customers, Order, Item, Sequelize } = require('./models');
+
+
+const app = express();
+const PORT = 4000;
+
+
+app.use(express.json());
+
+
+app.get('/customers', async (req, res) => {
+    const result = await Customers.findAll().catch((err) => {
+        res.status(500).send(`something wnt wrong ${err}`);
+    })
+
+    res.status(200).send(result);
+})
+
+app.post('/customers', async (req, res) => {
+    const { firstName, lastName, email, password, address } = req.body;
+    const result = await Customers.create({
+        firstName: firstName,
+        lastName: lastName,
+        email: email,
+        password: password,
+        address: address
+    }).catch((err) => {
+        res.status(400).send(`something went wrong ${err}`)
+    })
+    res.status(200).send(result);
+})
+
+
+app.delete('/customers/:customer_id', async (req, res) => {
+    const removalCustomer = await Customers.findAll({
+        where: {
+            id: req.params.customer_id
+        }
+    }).catch((err) => {
+        res.status(500).send(`something went wrong ${err}`)
+    })
+
+    await Customers.destroy({
```

Comment:
Before destroying the data we can check if that customer exists in the db

```suggestion
    if (foundCustomer) {
     await Customers.destroy({
```